src/Certify.Tests/Certify.Core.Tests.Unit/CertifyManagerAccountTests.cs

```diff
@@ -250,7 +273,7 @@ public async Task TestCertifyManagerAddAccountMissingAccountUri()
                 AgreedToTermsAndConditions = true,
                 CertificateAuthorityId = "letsencrypt.org",
                 EmailAddress = contactRegEmail,
-                ImportedAccountKey = DotNetEnv.Env.GetString("RESTORE_KEY_PEM"),
+                ImportedAccountKey = DotNetEnv.Env.GetString("RESTORE_KEY_PEM")?.Replace("\\r", "\r")?.Replace("\\n", "\n")?.Replace("'", ""),
```

Comment:
Could we use a shared utility method (or methods) for getting config items so that DotNetEnv isn't directly referenced everywhere? I think we may change this to loading from a secrets store etc in the future.